site/docs/concepts/README.md

```diff
@@ -1,255 +1,232 @@
 
-A **catalog** comprises all the components that describe how your data flows function and behave:
-captures, collections, derivations, materializations, tests, and more. For example:
+There are two main environments in which you can work with Flow:
+the web application or your preferred local environment using the flowctl command-line tool.
 
-* How to **capture** data from source systems into **collections**
-* The **schemas** of those collections, which Flow enforces
-* How to **derive** collections as transformations of other source collections
-* **Materializations** of collections into destination systems
-* Your **tests** of schema and derivation behaviors
+### Web application
 
-Together the captures, collections, derivations, and materializations of your catalog
-form a graph of your data flows:
+The Flow web application is where you'll most likely create, monitor, and update your data flows.
+You can find it at [dashboard.estuary.dev](https://dashboard.estuary.dev).
 
-import Mermaid from '@theme/Mermaid';
+The application is designed for intuitive use, and only requires a few [basic concepts](#essential-concepts) to get started.
+For a complete walk-through, see the [guide to create a dataflow](../guides/create-dataflow.md).
 
-<Mermaid chart={`
-	graph LR;
-		capture/two-->collection/D;
-		capture/one-->collection/C;
-		capture/one-->collection/A;
-        collection/A-->derivation/B;
-        collection/D-->derivation/E;
-        collection/C-->derivation/E;
-        derivation/B-->derivation/E;
-		collection/D-->materialization/one;
-		derivation/E-->materialization/two;
-`}/>
+The app is backed by secure, cloud-hosted infrastructure that Estuary manages.
 
-### Namespace
+### flowctl
 
-All catalog entities (captures, materializations, and collections) are identified by a **name**
-such as `acmeCo/teams/manufacturing/anvils`. Names have directory-like
-prefixes and every name within Flow is globally unique.
+flowctl is Flow's command-line interface.
+With flowctl, developers connect to the Estuary servers and work directly on active catalogs and drafts created in the Flow webapp.
+They can also develop data flows locally, test more flexibly, and collaboratively refine catalogs.
 
-If you've ever used database schemas to organize your tables and authorize access,
-you can think of name prefixes as being akin to database schemas with arbitrary nesting.
+[Learn more about flowctl](flowctl.md)
 
-All catalog entities exist together in a single **namespace**.
-As a Flow customer, you're provisioned one or more high-level prefixes for your organization.
-Further division of the namespace into prefixes is up to you.
+## Essential concepts
 
-Prefixes of the namespace, like `acmeCo/teams/manufacturing/`,
-are the foundation for Flow's authorization model.
+*In this section, you'll find the most important Flow terms and concepts. If you're new to Flow, start here.*
 
-### Builds
+Data flows have three required components:
```

Comment:
I'm not sure that this breakdown between 'required' and 'optional' components add clarity. My 2 cents is that it'd be clearer to just list all 4 things and not mark any subset as being 'required'. The term 'required' makes it seem like we would error out if their catalog doesn't have a capture. Maybe just change the wording to be more like "Data flows typically have these components:"?

Reply:
I'm trying to distinguist the idea of a "data flow" (ie, the end result, a pipeline) from the notion of a catalog (the instructions you give Flow to build said data flow). As someone who doesn't naturally think in terms of config files, this makes sense to me: I am a user who wants to integrate data from source to destination with Flow. To create that, at minimum, there must be a capture, collection, and materialization. 

I do agree that the "required" and "optional" verbiage is a bit strong, though. I will mellow this out.